### PR TITLE
#6 Issue Adding heading and Fixing Typo

### PR DESCRIPTION
**Pull request by ghost**
Hi, These are several changes I made.

I put a comment in each section about the changes I made.

I hope this comes across just right. If you find it hard to understand, please kindly let me now.

Thank you!

### CODE REVIEW

**Comment by samipak458** [comment]
Hi,
Please add screen shots of your changes. So, I can see how website is looking visually.

Thank you.

**Comment by DHCJS** [comment]
Can you show me how to deliver the page of the index.html after I change the code and make it available for you to screenshot?

**Comment by samipak458** [comment]
> Do I screenshot my code or the actual live preview website?
> 
> If the code here it is. Choir
> 
> Hopefully this is what you are looking for

No, I already see the code. I want to see the output of your code. Somehow same thing done by one contributor before.

**Comment by ghost** [comment]
Is this how to do it?

1. Add a heading for the first section
image

2. Change the position of the radio button into alphabetically ordered
image

3. Change the border looks from dashed to solid 
image

**Comment by samipak458** [comment]
Hi,
Kindly sync the fork.

File: index.html [hunk 1]
```diff
@@ -19,14 +19,16 @@
         #clear {
             display: none;
         }
-
+/*Changed #libraryForm and table(just personal opinion)*/
         #libraryForm {
-            border: .5px dashed black;
```
Comment (ghost) [inline index.html:71]: Looks solid

**Comment by ghost** [comment]
Done @samipak458. Please kindly check it.

**Comment by samipak458** [comment]
Hi @DHCJS,
One of the contributor makes UI very eye catching. 
Thank you for contribution, I really appreciate that. You can still contribute by fixing the current UI.
Have a look: https://delightful-coast-00b51a700.1.azurestaticapps.net/

**Comment by ghost** [comment]
This is the best I can do right now.
Fix:

- Alphabetically order in type
Add:

- Add Your Book in the first section

image

Let me know what you think

**Comment by samipak458** [comment]
There is still conflict in index.html

File: index.html [hunk 2]
```diff
@@ -16,26 +16,96 @@
...
+       #clear {
+            display:none;
+            text-transform: uppercase;
+            font-size: small;
+            margin-bottom: 10px;
+            
+        
         }
-/*Changed #libraryForm and table(just personal opinion)*/
+     #clear:hover{
```
Comment (ghost) [inline index.html]: This is really great update from the last contributor

File: index.html [hunk 3]
```diff
@@ -16,26 +16,96 @@
...
             padding: 20px;
+	    border-radius:20px;
         }
 
-        #table {
-            border: 2px solid black;
-            border-radius: 15px;
-            padding: 10px;
+
+    .form-check1:hover,.form-check2:hover,.form-check3:hover,.form-check4:hover, .form-check5:hover, .form-check6:hover{
```
Comment (ghost) [inline index.html:75]: Make it more clear and pop out.

File: index.html [hunk 4]
```diff
@@ -70,58 +146,83 @@
 
     <!--Section of Adding books-->
     <div class="container pt-4 mt-5 my-3">
-        <!--How about change this into Header to make it more semantic layout? From header > h1 > h2-->
-    <header class="text-center display-4 font-weight-bold">Welcome To Online Library</header>
+        <h1 class="text-center display-4 font-weight-bold">Welcome To Online Library</h1>
         <hr>
-      <!--Section one Heading-->
-      <h1>Add a Book</h1>
+       <h1><font size="8px"><b>Add Your Book</b></font></h1>
```
Comment (ghost) [inline index.html:148]: This is what I add to the current contributor UI

File: index.html [hunk 5]
```diff
@@ -70,58 +146,83 @@
...
-                    Other
-                </label>
+            <div class="form-group row">
+                <label for="edition" class="col-sm-2 col-form-label">Edition:</label>
+                <div class="col-sm-10">
+                    <input type="number" class="form-control" id="edition" placeholder="Edition of Book">
+                </div>
             </div>
-        </div>
+            <div class="form-group row">
```
Comment (ghost) [inline index.html]: Changing few order

**Review by ghost — COMMENTED** [review]
Thank you for the learning opportunity

**Comment by samipak458** [comment]
Ok great, no issue with it.
Thanks for contributing

**Comment by ghost** [comment]
Thank you, @samipak458, for the learning opportunity. Hopefully, I can do better next time.

**Comment by samipak458** [comment]
Bro you have to resolve conflict because merge button is disabled.

**Comment by ghost** [comment]
Let me look into it.

Sorry for causing much trouble, bro.

**Comment by samipak458** [comment]
Yes, ok now 👍

**Pull request merged**